### DISCUSSION TITLE
Paper-桥边红药的论文 | Zi-Zi's Journey

### DISCUSSION THREAD

**Discussion by giscus** [GZ-Metal-Cell/GZ-Metal-Cell.github.io#346] [Announcements]
# Paper-桥边红药的论文 | Zi-Zi's Journey

拜读一下 pro 神的大作😭。

https://gz-metal-cell.github.io/2024/10/06/Paper-%E6%A1%A5%E8%BE%B9%E7%BA%A2%E8%8D%AF%E7%9A%84%E8%AE%BA%E6%96%87/

**Comment by promefire** [comment]
😨好几处错误😨

**Reply by GZ-Metal-Cell** [reply]
😭论文总体结构还是很完整的 欣赏完 pro 神的论文深刻感受到自身的不足
